definitions/npm/react-apollo_v2.x.x/flow_v0.58.x-/react-apollo_v2.x.x.js

```diff
@@ -21,6 +21,10 @@ declare module 'react-apollo' {
     client: ApolloClient;
   }
 
+  declare export class ApolloConsumer extends React$Component<{}> {
+    render(): (client: ApolloClient) => React$Element<*>;
```

Comment:
I *think* this should be `React$Node`, unless you have reasoning otherwise?

Reply:
Yes, a typo. But for now all this new definition is totally wrong :(
The ApolloConsumer render attribute should be a function that returns `React$Node`; and the render prop `(client: ApolloClient) => React$Node` should be defined on the `React$Component` first type argument.
Let me work on that